doc/translations/zh_CN.po

```diff
@@ -72797,16 +72797,16 @@ msgstr ""
 msgid ""
```

Comment:
This file is automatically generated and should not be edited, translations are managed using [Weblate](https://hosted.weblate.org/projects/godot-engine/).